feature_elicitator:
    ros__parameters:
        setup:
            prefix: 'j2s7s300_driver'
            model_filename: "jaco_dynamics"
            object_centers: {'HUMAN_CENTER': [-0.2,-0.5,0.6], 'LAPTOP_CENTER': [0.0,-0.8,0.0]}
            feat_list: ["table"]
            feat_weights: [0.0]
            start: [104.2, 151.6, 183.8, 101.8, 224.2, 216.9, 310.8]
            goal: [210.8, 101.6, 192.0, 114.7, 222.2, 246.1, 322.0]
            goal_pose: None #[-0.46513, 0.29041, 0.69497]
            T: 20.0
            timestep: 0.5
            save_dir: "/data/features/"
            # These constants have been tuned for the task tested.
            INTERACTION_TORQUE_THRESHOLD: [0.88414821, 17.22751856, -0.40134936,  6.23537946, -0.90013662, 1.32379884,  0.10218059]
            INTERACTION_TORQUE_EPSILON: [4.0, 5.0, 3.0, 4.0, 1.5, 1.5, 1.5]
            FEAT_RANGE: {'table':0.98, 'coffee':1.0, 'laptop':0.3, 'human':0.3, 'efficiency':0.22, 'proxemics': 0.3, 'betweenobjects': 0.2}
            LF_dict: {'bet_data': 5, 'sin': False, 'cos': False, 'rpy': False, 'lowdim': False, 'EErot': True, 'noxyz': False, 'norot': False, 'noangles': True, '6D_laptop': False, '6D_human': False, '9D_coffee': False}
            CONFIDENCE_THRESHOLD: 1.0
            N_QUERIES: 10
            nb_layers: 3
            nb_units: 128

        planner:
            # These settings have been tuned for trajopt planner.
            type: "trajopt"
            max_iter: 50
            num_waypts: 5

        controller:
            # These settings have been tuned for PID controller.
            type: "pid"
            p_gain: 50.0
            i_gain: 0.0
            d_gain: 20.0
            epsilon: 0.10
            max_cmd: 40.0

        learner:
            # These settings have been tuned for beta learner.
            type: "max"
            step_size: 1.0
            alpha: -0.01
            n: 5
            P_beta: {"table0": [1.83701582842, 0.0, 0.150583961407], "table1": [2.8, 0.0, 0.4212940611], "coffee0": [1.67451171875, 0.0, 0.05], "coffee1": [2.8169921875, 0.0, 0.3], "human0": [2.14693459432, 0.0, 0.227738059531], "human1": [5.0458984375, 0.0, 0.25]}


